srcpkgs/cross-x86_64-w64-mingw32/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'cross-x86_64-w64-mingw32'
-_gcc_version=9.3.0
-_binutils_version=2.32
-_gmp_version=6.1.2
-_mpfr_version=4.0.1
+_gcc_version=10.2.0
+_binutils_version=2.34
+_gmp_version=6.2.0
+_mpfr_version=4.0.2
```

Comment:
The gcc 10.2 PR is using mpfr 4.1.0, from what I can see. Should we perhaps use that instead?

Reply:
69e1505